tools/proofers/for_mentors.php

```diff
@@ -185,7 +185,7 @@ function output_project_details($mentored_round, $projectid, $nameofwork, $autho
 
 function get_beginner_projects_in_state($state, $mentored_round)
```

Comment:
This is a good time to add types to this function def so phpstan can help us enforce this change.